### PR TITLE
Ft train premptive node

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
- Adds infrastructure support needed to run the train job once every month
- Updates the train job to run monthly instead of weekly
- Automates the adding and removal of the monthly train job

**_WHAT ISSUES ARE RELATED TO THIS PR?_**
- Automation works

**_HOW DO I TEST OUT THIS PR?_**
- You may analyse the code
- You may also observe the training cronjob running on the first day of the month between 2pm and 4pm EAT

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
- n/a

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
- n/a

**_ARE THERE ANY RELATED PRs?_**
- n/a

### CODE REVIEW

File: k8s/predict/train-job/values-prod.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ app:
   namespace: production
   configmap: env-train-job-production
 
-schedule: 0 5 * * 1
+schedule: 0 1 1 * *
```
Comment (123MwanjeMike) [inline k8s/predict/train-job/values-prod.yaml:6]: [Schedule train job to run on the 1st of every month at 01:00 UTC](https://github.com/airqo-platform/AirQo-api/pull/1333/commits/ba10ed140f54b9fc1671a4f422a7798d5fc86499)

File: k8s/nodes/airqo-stage-k8s-worker-2.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "kind": "Node",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "airqo-stage-k8s-worker-2",
+    "labels": {
+      "node-type": "general-purpose"
+    }
+  }
+}
```
Comment (123MwanjeMike) [inline k8s/nodes/airqo-stage-k8s-worker-2.json:10]: [rename node manifests](https://github.com/airqo-platform/AirQo-api/pull/1333/commits/b037d54b35786fb7e946c0562bf87d2a4f3c5c45)

File: infra/terraform/gcp/modules/ComputeDisk/airqo-k8s-worker-4.tf [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  image                     = var.os["ubuntu-focal"]
+  name                      = "airqo-k8s-worker-4"
+  physical_block_size_bytes = 4096
+  project                   = var.project_id
+  size                      = var.disk_size["medium"]
+  type                      = "pd-standard"
+  zone                      = var.zone
+  description               = "Disk for the airqo-k8s-worker-4 instance"
+}
+# terraform import google_compute_disk.airqo_k8s_worker_4 projects/${var.project_id}/zones/${var.zone}/disks/airqo-k8s-worker-4
```
Comment (123MwanjeMike) [inline infra/terraform/gcp/modules/ComputeDisk/airqo-k8s-worker-4.tf:11]: [airqo-k8s-worker-4 compute disk](https://github.com/airqo-platform/AirQo-api/pull/1333/commits/b1bfb55fd0e9d3f6440e8fa4a2ecbc920d297634)

File: infra/terraform/gcp/modules/ComputeInstance/airqo-k8s-worker-4.tf [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    source      = "https://www.googleapis.com/compute/v1/projects/airqo-250220/zones/europe-west1-b/disks/airqo-k8s-worker-4"
+  }
+
+  metadata = {
+    startup-script = "sudo ufw allow ssh"
+  }
+
+  network_interface {
+    access_config {
+      network_tier = "PREMIUM"
```
Comment (123MwanjeMike) [inline infra/terraform/gcp/modules/ComputeInstance/airqo-k8s-worker-4.tf:19]: [airqo-k8s-worker-4 compute instance](https://github.com/airqo-platform/AirQo-api/pull/1333/commits/8b34ca964c91cab3eddbd6881e54ba408dc817ae)

File: infra/terraform/gcp/modules/ComputeResourcePolicy/monthly-k8s-runners.tf [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    vm_start_schedule {
+      schedule = "40 0 1 * *"
+    }
+    vm_stop_schedule {
+      schedule = "0 3 1 * *"
+    }
+    time_zone = "UTC"
+  }
+}
+# terraform import google_compute_resource_policy.monthly_k8s_runners projects/${var.project_id}/regions/${var.region}/resourcePolicies/monthly-k8s-runners
```
Comment (123MwanjeMike) [inline infra/terraform/gcp/modules/ComputeResourcePolicy/monthly-k8s-runners.tf:16]: [add 'monthly_k8s_runners' instance schedule](https://github.com/airqo-platform/AirQo-api/pull/1333/commits/5a074a6df73416d28d0776282648f285f0ea15d0)

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Review by Baalmart — APPROVED** [review]

**Pull request merged**